common/bs_scrapers/get_files.py

```diff
@@ -53,7 +53,7 @@ def get_files(save_dir, sleep_time, delete=True, debug=False):
                     file_name = file_name + ".xls"
                 if os.path.exists(save_dir + file_name) == False:
                     try:
-                        pdf = urllib.request.urlopen(url_2)
+                        pdf = urllib.request.urlopen(url_2.replace(" ", "%20"))
```

Comment:
https://docs.python.org/3/library/urllib.parse.html#urllib.parse.quote would probably be more robust here

Reply:
For now, being that it works, that is Josh C's main priority. You can make a PR updating this either against my branch, or against the main branch. 👍